loader.js

```diff
@@ -113,10 +115,25 @@ define([
       });
     }
 
+    // install optional plugins, summary widget is handled separately as it was added long ago
     if (config.plugins) {
       if (config.plugins.summaryWidgets) {
         openmct.install(openmct.plugins.SummaryWidget());
       }
+
+      const pluginsToInstall = Object.keys(config.plugins).filter(
+        (plugin) => plugin !== 'summaryWidgets' && optionalPlugins.includes(plugin)
```

Comment:
why treat summaryWidgets special from other optionalPlugins?

Reply:
they aren't setup the same way, the idea here is have the name of the plugin match the name in `openmct.plugins[PluginName]`, summary widgets isn't matching with regard to case. I don't want to change it so it's still backwards compatible.